src/components/SideTray/SideTray.tsx

```diff
@@ -1,20 +1,54 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, ReactNode, CSSProperties } from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 
 // Rover UI dependencies
 import { parseCssSize } from '../../shared/css-utils';
 
 // This component's dependencies
 import styles from './SideTray.module.css';
 
-const SideTray = ({
+interface SideTrayProps {
+  children: ReactNode;
+  className?: string;
+  direction?: string;
```

Comment:
A suggestion to set limit the number of options you can pass to `direction`: like

```ts
type SideTrayDir = 't' | 'b' | 'r' | 'l';

interface SideTrayProps {
  // ...
  direction?: SideTrayDir;
  // ...
}
```

That way it's enforceable at a type level and not just run-time

Reply:
True, let me add this real quick and this will be all good.